crates/core/src/config/secret.rs

```diff
@@ -240,14 +240,14 @@ mod tests {
     use super::*;
 
     #[test]
-    fn test_load() {
+    fn test_load_from_secret_dir() {
```

Comment:
I think you are mixing up 2 different things. Secret dir are not for storing key, but for storing delegate secrets which is different! (Those may be keys or whatever private information in general). The keys is what we generate/load/default to encrypt those secrets. We don't load any key from secret dir neither store keys in there.

Reply:
isn't this test redundant with `test_load_from_different_files` anyways. let's just remove this and then is good